docs/basisterminologie.md

```diff
@@ -0,0 +1,184 @@
+
+Maar het onderwerp kan ook meer specifiek of persoonlijk zijn en betrekking hebben op een specifiek voor één klant gestart proces (“kunt u toelichting geven bij het verzoek extra informatie aan te leveren om de aanvraag van mijn omgevingsvergunning te kunnen beoordelen?”). Om in verschillende situaties onderwerp(en) van het klantcontact te kunnen vastleggen, worden onderwerpdelen geregistreerd. Net als de klantcontactinhoud, kunnen deze kunnen verschillende vormen hebben:
+
+1. (‘Platte’) tekst die (een deel van) het onderwerp van het klantcontact beschrijft.
+2. Een verwijzing naar de interne taak die aanleiding gaf tot (een deel van) het klantcontact.
+3. Een verwijzing naar de zaak waarover (een deel van) het klantcontactging.
+4. Een verwijzing naar de klanttaak waarover (een deel van) het klantcontact ging.
+
+<img src="./assets/basisterminologie/klantinteracties-submodel-onderwerp-klantcontact.svg" alt="Submodel onderwerp van het klantcontact" width="52%"/>
+
+### Betrokkene bij het klantcontact
+
+Zowel inzage in historische contacten met de gemeente als opvolging van verzoeken aan de gemeente maken het nodig bij te houden welke personen of organisaties bij klantcontacten betrokken waren. Daarvoor dient ‘betrokkene bij klantcontact’. Naast de rol die de betrokkene bij een klantcontact speelde (belanghebbende of vertegenwoordiger), kunnen hierin contactgegevens (in de vorm van zowel ‘fysieke’ als [‘digitale’](#digitaal-adres) adressen) worden vastgelegd. Deze dienen voor opvolging van het klantcontact.
+
+Als de betrokkene bij het klantcontact als [partij gekend](#partij) is, en ófwel opvolging van het klantcontact in het geheel niet nodig is, ófwel voor die opvolging de bij partij vastgelegde contactgegevens mogen worden gebruikt, blijven de contactgegevens bij ‘betrokkene bij klantcontact’ leeg. In twee andere gevallen worden bij ‘betrokkene bij klantcontact’ wél contactgegevens geregistreerd:
+
+1. Als de betrokkene bij het klantcontact als partij gekend partij is, maar deze partij wenst dat voor het opvolgen van het klantcontact eenmalig contactgegevens worden gebruikt die afwijken van de ‘default’contactgegevens die bij partij zijn geregistreerd.
+2. Als de betrokkene bij het klantcontact niet als partij gekend is, maar opvolging van het klantcontact registratie van (enige) contactgegevens noodzakelijk maakt.
+
+Als bij een contact geen enkele gekende partij betrokken is, én dat contact geen aanleiding geeft tot opvolging, dient het vastleggen van een klantcontact geen doel. Registratie kan in dit geval dus achterwege blijven.
```

Comment:
Deze opmerking begrijp ik niet zo goed. Als er een contact geweest is met een klant (of een vertegenwoordiger) die netjes naam etc heeft opgegeven, maar aangeeft dat de gegevens niet voor hergebruik mogen worden vastgelegd, spreken we dan wel van een gekende partij ? Dat contact wil je namelijk wel vastleggen volgens mij

Reply:
Je bedoelt regel 72? Die gaat uit van een situatie waarin onvoldoende authenticatie heeft plaatsgevonden om de persoon of organisatie waarmee de gemeente contact heeft betrouwbaar te identificeren.

In dit geval bestaat onvoldoende basis om de juiste bij het contact betrokken partij bij op te zoeken of een nieuwe partij te registreren. Zo'n situatie zal vaak voorkomen bij het beantwoorden van een eenvoudige, generieke vragen naar bijvoorbeeld de bereikbaarheid van een afdeling of de openingstijden van de gemeentewerf.

Omdat koppelen aan een partij niet geoorloofd is, kunnen we het verslag van zo'n contact niet tonen in een 'Mijn-portaal'. Als naar aanleiding van het contact óók geen opvolging nodig is, dient het vastleggen van een klantcontact dus geen doel meer. Volgens mij kun je je dan de moeite van registratie beter besparen.